source/js/utility.js

```diff
@@ -27,6 +27,18 @@ const Utility = {
     });
 
     return allFound ? elements : false;
+  },
+
+  /**
+   * Hide newsletter button on nav if the page's main content section has a sign up form already.
+   */
+  hideNavNewsletterButtonConditionally() {
```

Comment:
Did we look into whether we can do this at the Django level, rather than the JS level?

Reply:
I'll look into it.

Reply:
With Django approach, I can only test if a page has a signup snippet associated with it to determine if the nav button should be rendered or not. For example, in the future, if dev adds  `<div class="join-us"></div>` somewhere in the template level, a signup form will get injected via React, and Django won't know. This probably isn't a huge issue but I'll ask in the original ticket to double check.